karaf/features/src/main/feature/feature.xml

```diff
@@ -100,6 +100,38 @@
         <bundle dependency="true">mvn:javax.servlet/javax.servlet-api/${javax-servlet.version}</bundle>
     </feature>
 
+    <feature name="brooklyn-rest-client" version="${project.version}" description="Brooklyn REST Client">
+        <feature>brooklyn-rest-api</feature>
+        <feature>swagger</feature>
+        
+        <bundle>mvn:org.apache.brooklyn/brooklyn-rest-client/${project.version}</bundle>
+        
+        <bundle dependency="true">mvn:org.apache.httpcomponents/httpclient-osgi/${httpclient.version}</bundle>
+        <bundle dependency='true'>mvn:commons-logging/commons-logging/${commons-logging.version}</bundle>
+        <bundle dependency='true'>mvn:commons-codec/commons-codec/${commons-codec.version}</bundle>
+
+        <bundle dependency='true'>wrap:mvn:com.google.code.findbugs/jsr305/${jsr305.version}</bundle>
+
+        <bundle dependency='true'>mvn:com.google.code.gson/gson/${gson.version}</bundle>
+        
+        <bundle dependency='true'>wrap:mvn:org.jboss.resteasy/resteasy-jaxrs/${resteasy.version}</bundle>
+        <bundle dependency='true'>wrap:mvn:org.jboss.resteasy/resteasy-jackson2-provider/${resteasy.version}</bundle>
+        <bundle dependency='true'>wrap:mvn:org.jboss.resteasy/jaxrs-api/${resteasy.version}</bundle>
+        <bundle dependency='true'>mvn:org.jboss.spec.javax.annotation/jboss-annotations-api_1.1_spec/${jboss-annotations-api_1.1_spec.version}</bundle>
```

Comment:
The Felix framework already provides a `javax.annotation` package (bundle 0) at version 1.0. This one provides the same package at version 1.1. This could potentially lead to `Uses constraint violation` when resolving the bundles, depending on loading order. If possible better remove it. Resteasy being wrapped doesn't have a dependency on a specific version of the package.